incubator/redis-cache/templates/ss.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: "{{ template "fullname" . }}"
+  labels:
+     app: {{ template "fullname" . }}
```

Comment:
The app label should only have the name. Please fix everywhere.

`app: {{ template "name" . }}`